rgw/v2/tests/s3cmd/test_s3cmd.py

```diff
@@ -407,6 +410,27 @@ def test_exec(config, ssh_con):
             else:
                 log.info("bucket list is empty as expected")
 
+    elif config.test_ops.get("set_public_acl", False):
+        log.info("Verify setting public acl to the bucket doesn't result in error")
+        log.info("BZ2338149")
```

Comment:
@ckulal , include the BZ details as a comment, instead of logging it

Reply:
done